docs/operators/aws_cloudwatch_input.md

```diff
@@ -0,0 +1,140 @@
+| `id`                      | `aws_cloudwatch_input` | A unique identifier for the operator.                                                                       |
+| `output`                  | Next in pipeline       | The connected operator(s) that will receive all outbound entries.                                           |
+| `log_group_name`          | required               | The Cloudwatch Logs Log Group Name.                                                                         |
+| `region`                  | required               | The AWS Region to be used.                                                                                  |
+| `log_stream_name_prefix`  |                        | The log stream name prefix to use. This will find any log stream name in the group with the starting prefix |
+| `log_stream_names`        |                        | An array of log stream names to get events from.                                                            |
+| `profile`                 |                        | Profile to use for authentication. Details on named profiles can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) |
+| `event_limit`             | `10000`                | The maximum number of events to return per call.                                                            |
+| `poll_interval`           | `1m`                   | The duration between event calls.                                                                           |
+| `start_at`                | `end`                  | At startup, where to start reading events. Options are `beginning` or `end`                                 |
+
+### Log Stream Prefix
+
+The log_stream_prefix allows the use of "directives" such as `%Y` (4-digit year) and `%d` (2-digit zero-padded day). These directives are based on `strptime` directives. There are a limited set of the `strptime` directives. These directives are listed below. When directive is detected within the prefix it will replace the first occurance of directive with the data indicated in the description.
+
+#### Supported directives
+
+| Directive | Description                        |
+| :---:     | :---                               |
+| %Y        | Year, zero-padded                  |
+| %y        | Year, last two digits, zero-padded |
+| %m        | Month, zero-padded                 |
+| %q        | Month as a unpadded number         |
+| %b        | Abbreviated month name             |
+| %h        | Abbreviated month name             |
+| %B        | Full month name                    |
+| %d        | Day of the month, zero-padded      |
+| %g        | Day of the month, unpadded         |
+| %a        | Abbreviated weekday name           |
+| %A        | Full weekday name                  |
```

Comment:
It is not a feature of aws. I added it for cases where a new log stream name is created each day with new name. Similar behavior was in previous fluentd operator we used in past.  It isn't required to use this. If prefix or names is not provided it will get all events from that log group name.